packages/next/src/build/webpack/plugins/next-trace-entrypoints-plugin.ts

```diff
@@ -294,6 +361,20 @@ export class TraceEntryPointsPlugin implements webpack.WebpackPluginInstance {
     readlink: any,
     stat: any
   ) {
+    async function getOriginalHash(path: string): Promise<string> {
+      return new Promise((resolve, reject) => {
+        if (path.includes('?')) {
+          path = path.substring(0, path.indexOf('?'))
+        }
+        compilation.inputFileSystem.readFile(path, (err, result) => {
+          if (err) {
+            return reject(err)
+          }
+          resolve(getHash(result || ''))
```

Comment:
same question about empty string on this one

Reply:
should be the same as https://github.com/vercel/next.js/pull/65071#discussion_r1581301524